doc/api_break.dox

```diff
@@ -2431,6 +2431,10 @@ displayExpression instead. For the map tip use mapTipTemplate() instead.
 - addPart() now returns QgsGeometry::OperationResult enum, integer representation of returned values may have changed
 - splitParts() now returns QgsGeometry::OperationResult enum, integer representation of returned values may have changed
 - splitFeatured() now returns QgsGeometry::OperationResult enum, integer representation of returned values may have changed
+- pendingFields() was dropped. Use fields() instead.
+- pendingAllAttributesList() was dropped. Use allAttributes() instead.
+- pendingPkAttributesList() was dropped. Use pkAttributeList() instead.
```

Comment:
Can this be renamed to pkAttributes() for consistency with allAttributes?

Reply:
Done - I've renamed it to primaryKeyAttributes() to get rid of the acronym too.